src/jank/index.js

```diff
@@ -96,4 +104,16 @@ window.onload = () => {
       }
     );
   };
+
+  /**
+   * Redirects the user to the specified scorepad.
+   *
+   * @param {Object} scorepad the scorepad to redirect the user to
+   */
+  function redirectToScorepad(scorepad, sideName) {
+    window.location.href = `${sideName}?scorepad=${scorepad._id}`;
+  }
+  
+  scorepadList = document.getElementById('scorepad-list');
+  getScorepads('JanK', scorepadList, (scorepad) => {redirectToScorepad(scorepad, "jankSchreiben.html")});
```

Comment:
Und hier direkt ohne die `redirectToScorepad` Funktion:
```suggestion
  getScorepads('JanK', scorepadList, (scorepad) => {
    window.location.href = `jankSchreiben.html?scorepad=${scorepad._id}`
  });
```